lib/cisco_node_utils/vxlan_vtep_vni.rb

```diff
@@ -118,7 +114,7 @@ def remove_add_ingress_replication(protocol)
         config_set('vxlan_vtep_vni', 'ingress_replication', @set_args)
```

Comment:
Please include the fix I sent you via email for `def ingress_replication=(protocol)`

```
    def ingress_replication=(protocol)
      return if protocol.to_s == ingress_replication
      # Only set ingress_replicatin to the default value if it's not already.
      if protocol.to_s == default_ingress_replication &&
         (ingress_replication != default_ingress_replication)
        set_args_keys(state: 'no', protocol: ingress_replication)
        config_set('vxlan_vtep_vni', 'ingress_replication', @set_args)
      else
        # Multicast group and ingress replication are mutually exclusive
        # properties, so remove multicast_group if configured.
        unless multicast_group.empty?
          set_args_keys(state: 'no', ip_start: '', ip_end: '')
          config_set('vxlan_vtep_vni', 'multicast_group', @set_args)
        end
        remove_add_ingress_replication(protocol.to_s)
      end
    end

```